cmd/kured/main.go

```diff
@@ -84,131 +104,90 @@ func main() {
 	}
 }
 
-// newCommand creates a new Command with stdout/stderr wired to our standard logger
-func newCommand(name string, arg ...string) *exec.Cmd {
-	cmd := exec.Command(name, arg...)
+func isWindows() bool {
```

Comment:
I'm not sure of the expectations from the maintainers, but IMO splitting os-specific logic into separate files tends to be cleaner.

Reply:
Agreed. Even if it's ugly as hell, it's the way many go programs deal with windows. Though, I am certainly new in this ecosystem.